tutorials/tracing.md

```diff
@@ -0,0 +1,182 @@
+the collector we have to start the jaeger instance:
+
+```bash
+podman run -it --rm --network host jaegertracing/all-in-one:1.41.0
+```
+
+After jaeger is up and running we can start the OpenTelemetry collector:
+
+```bash
+podman run -it --rm --network host \
+    -v ./otel-collector-config.yaml:/etc/otel-collector-config.yaml \
+    otel/opentelemetry-collector:0.70.0 --config=/etc/otel-collector-config.yaml
+```
+
+The collector logs will indicate that the connection to Jaeger was successful:
+
+```text
+2023-01-26T13:26:22.015Z        info    jaegerexporter@v0.70.0/exporter.go:184  \
+    State of the connection with the Jaeger Collector backend \
+    {"kind": "exporter", "data_type": "traces", "name": "jaeger", "state": "READY"}
+```
+
+The Jaeger UI should be now available on http://localhost:16686
+
+It's now possible to start CRI-O with enabled tracing:
+
+```bash
+sudo crio --enable-tracing --tracing-sampling-rate-per-million 1000000
+```
+
+And when now running a CRI API call, for example by using [`crictl`][crictl]:
+[crictl]: https://github.com/kubernetes-sigs/cri-tools
+
+```bash
+sudo crictl ps
+```
+
+Then the OpenTelemetry collector will indicate that it has received new traces
+and spans, where the trace with the ID `1987d3baa753087d60dd1a566c14da31`
+contains the invocation for listing the containers via `crictl ps`:
+
+```text
+Span #2
+    Trace ID       : 1987d3baa753087d60dd1a566c14da31
+    Parent ID      :
+    ID             : 3b91638c1aa3cf30
+    Name           : /runtime.v1.RuntimeService/ListContainers
+    Kind           : Internal
+    Start time     : 2023-01-26 13:29:44.409289041 +0000 UTC
+    End time       : 2023-01-26 13:29:44.409831126 +0000 UTC
+    Status code    : Unset
+    Status message :
+Events:
+SpanEvent #0
+     -> Name: log
+     -> Timestamp: 2023-01-26 13:29:44.409324579 +0000 UTC
+     -> DroppedAttributesCount: 0
+     -> Attributes::
+          -> log.severity: Str(DEBUG)
+          -> log.message: Str(Request: &ListContainersRequest{Filter:&ContainerFilter{Id:,State:&ContainerStateValue{State:CONTAINER_RUNNING,},PodSandboxId:,LabelSelector:map[string]string{},},})
+          -> id: Str(4e395179-b3c6-4f87-ac77-a70361dd4ebd)
+          -> name: Str(/runtime.v1.RuntimeService/ListContainers)
+SpanEvent #1
+     -> Name: log
+     -> Timestamp: 2023-01-26 13:29:44.409813328 +0000 UTC
+     -> DroppedAttributesCount: 0
+     -> Attributes::
+          -> log.severity: Str(DEBUG)
+          -> log.message: Str(Response: &ListContainersResponse{Containers:[]*Container{},})
+          -> name: Str(/runtime.v1.RuntimeService/ListContainers)
+          -> id: Str(4e395179-b3c6-4f87-ac77-a70361dd4ebd)
+```
+
+We can see that there are `SpanEvent`s attached to the `Span`, carrying the log
+messages from CRI-O. The visualization of the trace, its spans and log messages
+can be found in Jaeger as well, under
+`http://localhost:16686/trace/1987d3baa753087d60dd1a566c14da31`:
+
```

Comment:
Mention that spans can be nested under kubelet's if it has tracing enabled

Reply:
Yep, good point. Added as suggested :+1: